### ISSUE TITLE
Is it possible to answer autonomously?

### ISSUE THREAD

**Issue by krsnaa** [bmuscede/HackQuack#1]
Is it possible to use the websocket connection to send the computed answer automatically, in effect, have device autonomously 'play the game'?

**Comment by bmuscede** [comment]
Yes, that's what I'm *eventually* working toward with the app being capable of starting at 3 and 9 to just playing the game for you.

Currently the roadmap is:
* Add more question solvers to improve its answering ability. Right now it has ~80% accuracy.
* Save previous questions and answers to test each question solver.
* Make the app more customizable 
* Have the app autoplay the game for you.

Take a look at the `experimental` branch to see some of those changes.
